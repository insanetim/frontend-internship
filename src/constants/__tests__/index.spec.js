import * as constants from '../index'

it('constants matches snapshot', () => {
  expect(constants).toMatchSnapshot()
})
