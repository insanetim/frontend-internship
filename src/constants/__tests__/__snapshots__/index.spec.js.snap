// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants matches snapshot 1`] = `
{
  "API_KEY": "57dbb283c800ccd0bc9a7352c7f5fa3b",
  "API_URL": "https://api.themoviedb.org/3/",
  "APP_NAME": "THE MOVIE DB",
}
`;
